### PR TITLE
Bump rascal 14

### PR DESCRIPTION
**Pull request by Betisman**
This is a bump of this component to the new versions of nodejs and rascal.

It now also uses async/await instead of callbacks.

The version is changing to a major one because the bumped libraries could have breaking changes and the users should migrate carefully.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,44 +1,57 @@
...
-    function validate(cb) {
-        if (!has(config, 'vhosts')) return cb(new Error('config.vhosts is required'))
-        cb()
-    }
+module.exports = options => {
 
-    function start(cb) {
-        logger.info('Connecting to rabbitmq')
-        var conf = RabbitClient.withDefaultConfig(config);
+  const RabbitClient = get(options, 'RabbitClient') || require('rascal');
```
Comment (w3dani) [inline index.js:5]: https://youmightnotneed.com/lodash

or 

https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_get

File: index.js [hunk 2]
```diff
@@ -1,44 +1,57 @@
...
+    config = { ...dependencies.config };
+    logger = ({ ...dependencies.logger }) || console;
+
+    if (!dependencies.logger) {
+      logger.info = console.log;
     }
+  };
+
+  const validate = () => {
+    if (!has(config, 'vhosts'))
```
Comment (w3dani) [inline index.js:20]: https://youmightnotneed.com/lodash

or 

https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_has

**Review by w3dani — COMMENTED** [review]
I think we should try remove lodash if we're using node 16

File: index.js (hunk 1 above)
Comment (Betisman) [reply index.js:5]: You're right, let's get rid of it

File: index.js (hunk 2 above)
Comment (Betisman) [reply index.js:20]: You're right, let's get rid of it

Comment (cressie176) [reply index.js:20]: No issues getting rid of lodash from systemic-rabbitmq, but Rascal will continue to use it so from that perspective it's kinda pointless

**Comment by UlisesGascon** [comment]
Deprecation in process. Please have a look to #12 @Betisman

**Pull request closed without merging**